src/main/webapp/app/exercises/shared/plagiarism/types/text/TextPlagiarismResult.ts

```diff
@@ -0,0 +1,19 @@
+import { TextComparison } from './TextComparison';
+import { PlagiarismResult } from '../PlagiarismResult';
+
+/**
+ * Result of the automatic plagiarism detection for text or programming exercises.
+ */
+export class TextPlagiarismResult extends PlagiarismResult {
+    comparisons: TextComparison[];
+
+    /**
+     * Duration of the plagiarism detection run in milliseconds.
+     */
+    duration: number;
```

Comment:
this could be interesting for modeling as well, please move it into the super class